TarponSimulator2017/Controller/MasterController.cs

```diff
@@ -23,6 +23,7 @@ public void ChangeScene (Scene newScene)
 
 		public void Update(GameTime gameTime, KeyboardState _keyboardState, KeyboardState _oldKeyboardState, MouseState mouseState)
 		{
+//			Console.WriteLine ("update MasterController");
```

Comment:
I think we can avoid commented debug statements in the committed code ?
To ease your tests, I recommend you the integrated debugger in monodevelop, which enable you to put breakpoints wherever you want without polluting your code 😜